atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1345,11 +1345,19 @@ bool WebContents::IsOffScreen() const {
 
 void WebContents::OnPaint(const gfx::Rect& dirty_rect,
                           const gfx::Size& bitmap_size,
+                          const int pixel_size,
                           void* bitmap_pixels) {
   v8::MaybeLocal<v8::Object> buffer = node::Buffer::New(
-      isolate(), reinterpret_cast<char*>(bitmap_pixels), sizeof(bitmap_pixels));
+      isolate(), reinterpret_cast<char*>(bitmap_pixels),
+      pixel_size * dirty_rect.width() * dirty_rect.height());
+
+  mate::Dictionary dict = mate::Dictionary::CreateEmpty(isolate());
+  dict.Set("width", bitmap_size.width());
+  dict.Set("height", bitmap_size.height());
+  dict.Set("pixel", pixel_size);
```

Comment:
But also it's much more common to have bits per pixel (8/16/24/32) not (1/2/3/4).